node/v2/args.js

```diff
@@ -23,6 +23,12 @@
 var TypedError = require('error/typed');
 var inherits = require('util').inherits;
 var bufrw = require('bufrw');
+var Checksum = require('./checksum');
+
+var Flags;
+process.nextTick(function() {
+    Flags = require('./index').CallFlags;
+});
```

Comment:
Would it be less smelly to just move this import to the bottom of the file, or even simply lazy-bind the index module?